kotlin-native/runtime/src/main/kotlin/kotlin/native/Platform.kt

```diff
@@ -101,6 +101,13 @@ public object Platform {
     public val isFreezingEnabled: Boolean
         get() = Platform_isFreezingEnabled()
 
+    /**
+     * Representation of the name used to invoke the program executable.
+     * [null] if the Kotlin code was compiled to a native library and the executable is not a Kotlin program.
+     */
+    public val programName: String?
+        get() = Platform_getProgramName()
```

Comment:
@qurbonzoda could you please review the stdlib change?

Reply:
LGTM